R/pkg/R/generics.R

```diff
@@ -799,6 +799,10 @@ setGeneric("write.df", function(df, path = NULL, ...) { standardGeneric("write.d
 #' @export
 setGeneric("randomSplit", function(x, weights, seed) { standardGeneric("randomSplit") })
 
+#' @rdname broadcast
+#' @export
+setGeneric("broadcast", function(x) { standardGeneric("broadcast") })
```

Comment:
> this list is sorted alphabetically within this section

Looks like it used to be at some point, but these days are long gone. I can reorder it right now, but this means rearranging a whole section.